wallpaper-bindings/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use more_wallpapers::Mode;
+
+pub fn set_wallpaper_from_file(file_path: &str) {
+    set_wallpaper(file_path)
+}
+
+#[tauri::command]
+fn set_wallpaper(file_path: &str) {
+    println!("Setting wallpaper from file: {}", file_path);
+    let images = vec![file_path];
+
+    let _ = more_wallpapers::set_wallpapers_from_vec(images, "", Mode::Crop);
+}
```

Comment:
This should go in `src-tauri/src/api.rs` so we don't need to have a dependency on `tauri` from this crate.

That Tauri command function should also check if the file is in our application storage root, like `delete_resource` does: https://github.com/mrjones2014/jdt/pull/10/files#diff-efb77b88c8f55a4c87cafaeb27b1e4a512258e0b9ffd8cc4a7daab8bbc9ed302R38

Reply:
Actually we'll wanna think about how to select which actual wallpaper image to use, so we can just drop the Tauri command for now and figure that out in a separate PR. We'll need some sort of algorithm for choosing an image (maybe at random or something) from all the installed repositories.